privacy-dashboard/privacy-dashboard-impl/src/main/java/com/duckduckgo/privacy/dashboard/impl/ToggleReportsFeature.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2024 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.privacy.dashboard.impl
+
+import com.duckduckgo.anvil.annotations.ContributesRemoteFeature
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.feature.toggles.api.Toggle
+
+@ContributesRemoteFeature(
+    scope = AppScope::class,
+    featureName = "toggleReports",
+    settingsStore = ToggleReportsFeatureSettingsStore::class,
+)
+interface ToggleReportsFeature {
+    @Toggle.DefaultValue(false)
+    fun self(): Toggle
+}
+
+fun ToggleReportsFeature.isEnabled(): Boolean = self().isEnabled()
+
+fun ToggleReportsFeature.getSettings(): String? = self().getSettings()
```

Comment:
We should not do this, it's dangerous. If we ever add a sub-feature to `toggleReports` we'll likely miss that calling isEnabled() on the sub-feature actually calls the top-level feature.

Reply:
Removed as part of migration to remove settingsstore